### PR TITLE
Added peer review seeding

### PR DESCRIPTION
**Pull request by WChrisK**

### CODE REVIEW

File: lib/tasks/peer_review.rake [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# NOTE: The assignment cannot be generated in the assignments rake task since
```
Comment (david-yz-liu) [inline lib/tasks/peer_review.rake]: @WChrisK Investigate this error further. This type of comment is too vague to belong.

File: lib/tasks/peer_review.rake [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# it causes errors to occur. I guess this is due to it not being populated
+# properly and something errors out. Therefore, all the peer review stuff is
+# done here, including but not limited to creating the PR assignment.
+
+namespace :db do
+  desc 'Create peer reviews and related data'
+  task :peer_reviews => :environment do
+    puts 'Creating A1 Peer Review and updating A1 to have the peer review'
+    a1 = Assignment.find_by(short_identifier: 'A1')
+    a1.has_peer_review = true  # Creates 'a1pr' via callback.
```
Comment (david-yz-liu) [inline lib/tasks/peer_review.rake]: It's the saving which runs the callback. You should use the `update` instead.

File: lib/tasks/peer_review.rake [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    a1.save
+
+    a1pr = Assignment.find_by(short_identifier: a1.short_identifier + '_pr')
```
Comment (david-yz-liu) [inline lib/tasks/peer_review.rake]: Better to do `a1pr = a1.peer_review_assignment`, or whatever the association is called.

File: lib/tasks/peer_review.rake [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    a1pr.clone_groupings_from(a1.id)
+
+    # TODO - Make a proper class for this so the code is clearer? Rather than some arbitrary list index?
+    # Populate a list of 'list of [group_id, result]'.
+    list_of_list_of_results_group_id = []
```
Comment (david-yz-liu) [inline lib/tasks/peer_review.rake]: It's not necessary to make this list.

Keep the second `each`, and inside that pick a random grouping from `a1.groupings`, and make the PeerReview.

**Comment by WChrisK** [comment]
Looks much nicer now IMO

**Comment by david-yz-liu** [comment]
Indeed it does.

**Pull request merged**